api/types/container/config.go

```diff
@@ -34,29 +34,29 @@ type HealthConfig struct {
 // All fields added to this struct must be marked `omitempty` to keep getting
 // predictable hashes from the old `v1Compatibility` configuration.
 type Config struct {
-	Hostname        string                // Hostname
```

Comment:
confused as to why spacing changed here, does not seem to be changed in this PR.